### ISSUE TITLE
Uncaught NotFoundError: Failed to execute 'setStartAfter' on 'Range

### ISSUE THREAD

**Issue by chrism** [emberjs/ember-inspector#260]
Hi,

When I switch to the Ember Inspector now & switch back to my console I notice that i'm now getting this error message.

`Uncaught NotFoundError: Failed to execute 'setStartAfter' on 'Range': The node provided is null. `

This is my setup:

Ember Inspector 1.6.2
Ember 1.8.1
Ember Data 1.0.0-beta.11
Handlebars 1.3.0
jQuery 1.11.1

It doesn't seem to affect the application (still works) but am always a bit concerned when I see red output in the console.

I can't make much sense of the error beyond it always seems to originate from this code in the VM file

```
    function virtualRange(view) {
      var start, end;
      var morph = view.get('morph');

      if (morph) {
        start = $('#' + morph.start)[0];
        end = $('#' + morph.end)[0];
      } else {
        // Support for metal-views
        morph = view.get('_morph');
        start = morph.start;
        end = morph.end;
      }

      var range = document.createRange();
      range.setStartAfter(start);
      range.setEndBefore(end);

      return range;
    }
```

**Comment by Little-Jon** [comment]
I just started having the same issue - It started showing up when I began working with creating and saving records with hasMany relationships with Ember Data. Not sure if that helps any, but I thought I'd mention it! The error's pretty inconsistent though.

**Comment by oskarrough** [comment]
I can second what @Little-Jon said, experiencing exactly the same. Seems to cause my page to get stuck in loading state. Can't always reproduce it. On Ember 1.9.1 and ember-data beta 0.14.1

**Comment by teddyzeenny** [comment]
I'd love to fix this before the next release this weekend. There's also a good chance the bug is in Ember core. If someone can somehow provide me with an online reproduction of the bug, I'll make sure to fix it asap.
@tomdale if I remember correctly you encountered the same issue a while back?

**Comment by tomdale** [comment]
In my experience, this is always related to malformed HTML in templates that the browser silently swallows but it breaks Ember's ability to later find those elements 
In our case, the fix was to switch to HTMLbars, which (embarrassingly) caught many cases of malformed HTML in our app, such as mismatched tags, missing closing tags, etc.

**Comment by Little-Jon** [comment]
Thanks @tomdale - I'll double-check my templates to make sure I didn't fat-finger some code. (:

**Comment by knownasilya** [comment]
I'm using 1.10 and htmlbars and get this error.

Seems like this method is using morphs?

```js
function getViewRange(view) {
      var range = document.createRange();
      range.setStartAfter(view._morph.start); <-- which is where `start` is null
      range.setEndBefore(view._morph.end);
      return range;
    }
```

**Comment by g-cassie** [comment]
@knownasilya Are you using liquid-outlet? I was getting the same error in 1.10 with htmlbars.  It happened any time you hover over a view for highlighting in the view tree if the view is a direct child of application.  When I changed the {{liquid-outlet}} to {{outlet}} in application.hbs the error stopped.

**Comment by knownasilya** [comment]
@g-cassie yes I am using liquid-outlet, but removing it is not an option :wink:

**Comment by locks** [comment]
Fixed by #428
